tests/files/line-numbers.css

```diff
@@ -0,0 +1,28 @@
+@charset "utf-8"; /* line 3 */
+
+@namespace "http://toto.example.org"; /* line 1 */
+
+@font-face { /* line 5 */
+    font-family: "CrassRoots";
+    src: url("http://example.com/media/cr.ttf")
+}
+
+
+#header { /* line 11 */
+    margin: 10px 2em 1cm 2%;
+    font-family: Verdana, Helvetica, "Gill Sans", sans-serif;
+    color: red !important;
+}
+
+@keyframes mymove {    /* line 17 */
+    from { top: 0px; } /* line 18 */
+
+    to { top: 200px; } /* line 20 */
+}
+
+@IMPORT uRL(test.css); /* line 23 */
+
+body {
+    background: #FFFFFF url("http://somesite.com/images/someimage.gif") repeat top center; /* line 25 */
+    color: rgb(233, 100, 450);
```

Comment:
Could you maybe split up the color components into separate lines and check if they are identified correctly:

```css
color: rgb( /* line 26 */
       233, /* line 27 */
       100, /* line 28 */
       450 /* line 29 */
);
```